<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>795</width>
    <height>596</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CipherGuard</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>791</width>
      <height>591</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>查询</string>
     </attribute>
     <widget class="QPushButton" name="delete_Button">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>380</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>删除</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string/>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>200</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>账 号：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>280</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>密 码：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>140</y>
        <width>54</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>备 注：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="query_Button">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>380</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>查询</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="user_ID">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>200</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="pass_WORD">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>270</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>450</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.5)</string>
      </property>
     </widget>
     <zorder>listWidget</zorder>
     <zorder>delete_Button</zorder>
     <zorder>comboBox</zorder>
     <zorder>label_5</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>query_Button</zorder>
     <zorder>user_ID</zorder>
     <zorder>pass_WORD</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>存储</string>
     </attribute>
     <widget class="QPushButton" name="create_btn">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>生成</string>
      </property>
     </widget>
     <widget class="QPushButton" name="submit_btn">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>420</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>提交</string>
      </property>
     </widget>
     <widget class="QWidget" name="gridLayoutWidget">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>351</width>
        <height>271</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>备注</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="pwdlength"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="userid"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>密码长度</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>楷体</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>账号</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="passwd"/>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="notes">
         <property name="minimumSize">
          <size>
           <width>10</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="hideorshow_btn">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>icon/preview-close-one.png</normaloff>icon/preview-close-one.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>70</y>
        <width>450</width>
        <height>400</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 0.5)</string>
      </property>
     </widget>
     <zorder>listView</zorder>
     <zorder>create_btn</zorder>
     <zorder>submit_btn</zorder>
     <zorder>gridLayoutWidget</zorder>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
